pushHandler.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"net/http"
+	"os"
+
+	guuid "github.com/google/uuid"
+	"gopkg.in/go-playground/webhooks.v5/github"
+)
+
+const (
+	tempRepoPath = "./temp_repo"
+)
+
+type pushHandler struct {
+	worker  []string
+	counter int
+}
+
+func (h *pushHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
Nit: This function is buggy if we get a push before the previous deployment finishes
Add a TODO comment, don't bother to fix for this PR